pkg/crc/adminhelper/manpages.go

```diff
@@ -0,0 +1,162 @@
+package adminhelper
```

Comment:
why the manpages are part of `adminhelper` package? `adminhelper` have different usecase so may be create a separate `manpages` package and keep those in that package?